src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -216,6 +241,13 @@ export const ComboBoxExample = {
           ability to specify a custom value in addition to selecting from a
           predetermined list.
         </p>
+        <p>
+          You must add an accessible label to each instance of{' '}
+          <strong>EuiComboBox</strong>. Labels can be created by wrapping the
+          combobox in an <strong>EuiFormRow</strong>, adding an{' '}
+          <EuiCode language="js">aria-label</EuiCode> prop, or passing a text
+          node ID to the <EuiCode language="js">aria-labelledby</EuiCode> prop.
+        </p>
```

Comment:
I think we should make this an `EuiCallout` with the `accessibility` icon. We do this in several places, for example, the flyout docs:

<img width="734" alt="" src="https://user-images.githubusercontent.com/549407/141335381-81595556-e6db-4f68-b8aa-9c3245bfb75f.png">

It looks like we also have these accessibility callouts in (e.g.) Pagination and Breadcrumbs, although those are yellow instead of blue - between the two I think I prefer blue/info.

```suggestion
        <EuiCallOut
          iconType="accessibility"
          title={
            <>
              You must add an accessible label to each instance of{' '}
              <strong>EuiComboBox</strong>
            </>
          }
        >
          Labels can be created by wrapping the combo box in an{' '}
          <strong>EuiFormRow</strong>, adding an <EuiCode>aria-label</EuiCode>{' '}
          prop, or passing a text node ID to the{' '}
          <EuiCode>aria-labelledby</EuiCode> prop.
        </EuiCallOut>
```